Tests/KlaviyoSwiftTests/NetworkSessionTests/testCreateEmphemeralSesionHeaders.1.txt

```diff
@@ -0,0 +1,20 @@
+▿ Optional<Dictionary<AnyHashable, Any>>
```

Comment:
Should this file and the one below be in a `__snapshots__` folder? They look like more snapshots anyway...

Reply:
No this got added by accident I think good catch!